TransformationSystem/Client/Utilities.py

```diff
@@ -342,7 +342,10 @@ def getPluginParam( self, name, default = None ):
     self.logDebug( "Transformation plugin param %s: '%s'" % ( name, value ) )
     if valueType and type( value ) != valueType:
       if valueType == type( [] ):
-        value = ast.literal_eval( value )
+        try:
+          value = ast.literal_eval( value )
+        except ValueError:
+          value = value.replace( ' ', '' ).split( ',' )
```

Comment:
<nitpicker>
In [13]: timeit.timeit( " 'dada didi'.strip(' ')", number = 10000000)
Out[13]: 1.5446979999542236

In [14]: timeit.timeit( " 'dada didi'.replace(' ','')", number = 10000000)
Out[14]: 2.15854811668396
</nitpicker>

:D but right